dask_kubernetes/operator/daskcluster.py

```diff
@@ -237,9 +245,15 @@ async def daskworkergroup_update(spec, name, namespace, logger, **kwargs):
             )
         logger.info(f"Scaled worker group {name} up to {spec['replicas']} workers.")
     if workers_needed < 0:
-        for i in range(current_workers, desired_workers, -1):
+        nest_asyncio.apply()
+        with rpc("localhost:8786") as scheduler:
```

Comment:
This should be the scheduler service name, not `localhost`.

Reply:
The argument the workers use is `f"tcp://{scheduler_name}.{namespace}:8786"` but that  failed.

Reply:
That is what I expected to be used. I think it might fail in development and when running tests because kopf is running outside the cluster. But in practice kopf will be inside the cluster and able to connect. I'm not immediately sure what the right thing is to do here.

Reply:
I'll add a TODO for it.